packages/client/subscriptions/NotificationSubscription.ts

```diff
@@ -91,6 +104,9 @@ const subscription = graphql`
       ...NotificationSubscription_meetingStageTimeLimitEnd @relay(mask: false)
       ...NotificationSubscription_paymentRejected @relay(mask: false)
 
+      # Generic notifications
+      ...NotificationSubscription_newNotification @relay(mask: false)
```

Comment:
+1 why `newNotification` instead of `notification` like the others?

Reply:
Removed this, as with the recent changes it is not needed